umm-spec-lib/src/cmr/umm_spec/migration/version/service.clj

```diff
@@ -35,10 +48,8 @@
   [context s & _]
   (-> s
       (assoc :AccessConstraints (first (:AccessConstraints s)))
-      (update :AccessConstraints #(util/trunc % 1024))
-      (update :UseConstraints #(util/trunc % 1024))
-      (update-in [:ServiceQuality :Lineage] #(util/trunc % 100))
       (assoc :RelatedURLs [(:RelatedURL s)])
+      (assoc :UseConstraints (first (:UseConstraints s)))
```

Comment:
I don't think the UMM-S V1.1 schema is correct in making `AccessConstraints` and `UseConstraints` a single type. It should be an array type. We should talk to Simon or Erich to get it fixed.

Reply:
These are single types in UMM-C as well:
    "UseConstraints": {
      "description": "Designed to protect privacy and/or intellectual property by allowing the author to specify how the collection may or may not be used after access is granted. This includes any special restrictions, legal prerequisites, terms and conditions, and/or limitations on using the item. Providers may request acknowledgement of the item from users and claim no responsibility for quality and completeness. Note: Use Constraints describe how the item may be used once access has been granted; and is distinct from Access Constraints, which refers to any constraints in accessing the item.",
      "$ref": "umm-cmn-json-schema.json#/definitions/UseConstraintsType"
    },
    "AccessConstraints": {
      "description": "Allows the author to constrain access to the collection. This includes any special restrictions, legal prerequisites, limitations and/or warnings on obtaining collection data. Some words that may be used in this element's value include: Public, In-house, Limited, None. The value field is used for special ACL rules (Access Control Lists (http://en.wikipedia.org/wiki/Access_control_list)). For example it can be used to hide metadata when it isn't ready for public consumption.",
      "$ref": "umm-cmn-json-schema.json#/definitions/AccessConstraintsType"
    },

It is the same there as it is in UMM-S here, where they are not arrays.  They are plural though because of English grammar not because they should be arrays.

Reply:
We should change `AccessConstraints` to `AccessConstraint`, `UseConstraints` to `UseConstraint` in both UMM-C and UMM-S. I am going to approve this PR as we don't want to block EDSC, but we should be consistent on our naming convention in UMM schemas.